scanner/node.go

```diff
@@ -12,6 +12,9 @@ func configureNode(sourceDir string, config *ScannerConfig) (*SourceInfo, error)
 		return nil, nil
 	}
 
+	remix := checksPass(sourceDir, dirContains("package.json", "remix"))
```

Comment:
Since we're deprecating the `remix` package won't be available. It would be better to check against `@remix-run/dev`

```suggestion
	remix := checksPass(sourceDir, dirContains("package.json", "@remix-run/dev"))
```

Reply:
We could also keep what we had, as that way we don't rely on package names

```suggestion
	remix := checksPass(sourceDir, fileExists("remix.config.js"))
```

If we do this, I would suggest to also check for the `*.prisma` file again 🤔

```go
	prisma := checksPass(sourceDir, fileExists("*.prisma"))
```

Reply:
What dirContains does is look for the string in the file, basically like grep.  So remix may produce false positives, but would match remix-run too.  I'm fine either way.

Reply:
Hmmm I think I would want to check for either `remix` or `@remix-run/dev` (since they're some sort of glob) in `package.json` as you technically could have no `remix.config.js` file and just use all basic config options 🤔

I do have a slight preference of `@remix-run/dev` though, but don't want to fight for it either 🤷‍♂️

Reply:
I probably didn't say something clearly.  But any package.json that has `@remix-run/dev` in it clearly also has the string `remix` in it.